x-pack/plugins/enterprise_search/server/routes/app_search/credentials.test.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { MockRouter, mockConfig, mockLogger } from '../../__mocks__';
```

Comment:
V small newline nit suggestion, mostly for consistency w/ other files

```suggestion
 */
 
import { MockRouter, mockConfig, mockLogger } from '../../__mocks__';
```